<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>lockTypeDlg</class>
 <widget class="QDialog" name="lockTypeDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>332</width>
    <height>454</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>锁类型定义</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../mainwindow.qrc">
    <normaloff>:/image/config.png</normaloff>:/image/config.png</iconset>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>厂站选择</string>
   </property>
  </widget>
  <widget class="QComboBox" name="stationCombo">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>20</y>
     <width>251</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>47</y>
     <width>311</width>
     <height>241</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="title">
    <string>已有锁类型</string>
   </property>
   <widget class="QTableWidget" name="lockTypeTable">
    <property name="geometry">
     <rect>
      <x>6</x>
      <y>25</y>
      <width>300</width>
      <height>211</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="modifyBtn">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>391</y>
     <width>75</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>修改</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>300</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>锁名称</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>332</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>锁类型</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>361</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>锁属性</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lockNameEdit">
   <property name="geometry">
    <rect>
     <x>61</x>
     <y>299</y>
     <width>251</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lockTypeEdit">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>330</y>
     <width>251</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lockAttrEdit">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>359</y>
     <width>251</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="addBtn">
   <property name="geometry">
    <rect>
     <x>126</x>
     <y>391</y>
     <width>75</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>添加</string>
   </property>
  </widget>
  <widget class="QPushButton" name="delBtn">
   <property name="geometry">
    <rect>
     <x>245</x>
     <y>391</y>
     <width>75</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>删除</string>
   </property>
  </widget>
  <widget class="QPushButton" name="loadDefaultBtn">
   <property name="geometry">
    <rect>
     <x>67</x>
     <y>423</y>
     <width>75</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>恢复默认值</string>
   </property>
  </widget>
  <widget class="QPushButton" name="saveDefaultBtn">
   <property name="geometry">
    <rect>
     <x>196</x>
     <y>423</y>
     <width>75</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>存为默认值</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../mainwindow.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>modifyBtn</sender>
   <signal>clicked()</signal>
   <receiver>lockTypeDlg</receiver>
   <slot>modifyLockType()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>40</x>
     <y>410</y>
    </hint>
    <hint type="destinationlabel">
     <x>43</x>
     <y>427</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>addBtn</sender>
   <signal>clicked()</signal>
   <receiver>lockTypeDlg</receiver>
   <slot>addLockType()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>160</x>
     <y>402</y>
    </hint>
    <hint type="destinationlabel">
     <x>216</x>
     <y>396</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>delBtn</sender>
   <signal>clicked()</signal>
   <receiver>lockTypeDlg</receiver>
   <slot>deleteLockType()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>301</x>
     <y>409</y>
    </hint>
    <hint type="destinationlabel">
     <x>311</x>
     <y>419</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>loadDefaultBtn</sender>
   <signal>clicked()</signal>
   <receiver>lockTypeDlg</receiver>
   <slot>loadDefault()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>115</x>
     <y>439</y>
    </hint>
    <hint type="destinationlabel">
     <x>160</x>
     <y>430</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>saveDefaultBtn</sender>
   <signal>clicked()</signal>
   <receiver>lockTypeDlg</receiver>
   <slot>saveDefault()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>255</x>
     <y>435</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>436</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>modifyLockType()</slot>
  <slot>addLockType()</slot>
  <slot>deleteLockType()</slot>
  <slot>loadDefault()</slot>
  <slot>saveDefault()</slot>
 </slots>
</ui>
